setup.py

```diff
@@ -19,23 +19,23 @@
 
 
 def main():
-    base_dir = dirname(__file__)
     setup(
         name="pytest-aviator",
-        version="0.1.1",
+        version="0.1.3",
```

Comment:
should i keep this or change to 0.0.1 for the actual plugin?

Reply:
I'm fine either way. Let's change if it doesn't impact anything. Or we can just keep `0.1.3` if it makes our lives easier to keep test and prod pypi in sync.